packages/client/hooks/useCallbackRef.ts

```diff
@@ -0,0 +1,17 @@
+import {MutableRefObject, useCallback, useRef} from 'react'
+import useForceUpdate from './useForceUpdate'
+
+const useCallbackRef = (): [(node) => void, MutableRefObject<HTMLDivElement | null>] => {
```

Comment:
We need to use the `phaseRef` width to calculate the initial number of subColumns when the page renders. It looks like a callback ref is the best way of doing this but I don't love the solution as reassigning a ref and using `forceUpdate` feels a lil hacky